azure/packages/azure-local-service/package.json

```diff
@@ -10,38 +10,13 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
-	"type": "commonjs",
-	"exports": {
-		".": {
-			"default": {
-				"types": "./dist/index.d.ts",
-				"default": "./dist/index.js"
-			}
-		},
-		"./alpha": {
-			"types": "./dist/azure-local-service-alpha.d.ts",
-			"default": "./dist/index.js"
-		},
-		"./beta": {
-			"types": "./dist/azure-local-service-beta.d.ts",
-			"default": "./dist/index.js"
-		},
-		"./internal": {
-			"types": "./dist/index.d.ts",
-			"default": "./dist/index.js"
-		},
-		"./public": {
-			"types": "./dist/azure-local-service-public.d.ts",
-			"default": "./dist/index.js"
-		}
-	},
-	"main": "dist/index.js",
-	"types": "dist/index.d.ts",
-	"bin": "dist/index.js",
+	"type": "module",
+	"bin": "lib/index.js",
 	"scripts": {
 		"build": "fluid-build . --task build",
 		"build:compile": "fluid-build . --task compile",
-		"build:docs": "api-extractor run --local",
+		"build:docs": "fluid-build . --task api",
```

Comment:
There is no api task. Also don't need api-extractor files.

Reply:
@jason-ha, @tylerbutler - policy-check won't let me remove build:docs...  so add an api task?

Reply:
I think others have that - or you can add an exception.
api-extractor:commonjs can't remain.

Reply:
In folder api-extractor files are still sitting around.